inference-engine/tests/functional/inference_engine/local_test.cpp

```diff
@@ -15,183 +15,636 @@ using namespace InferenceEngine;
 class LocaleTests : public ::testing::Test {
     std::string originalLocale;
     std::string _model = R"V0G0N(
-<net name="Power_Only" version="3" precision="FP32" batch="1">
-    <layers>
-        <layer name="data" type="Input" precision="FP32" id="0">
-            <output>
-                <port id="0">
-                    <dim>2</dim>
-                    <dim>3</dim>
-                    <dim>5</dim>
-                    <dim>5</dim>
-                </port>
-            </output>
-        </layer>
-        <layer name="power" type="Power" precision="FP32" id="1">
-            <data scale="0.75"
-                  shift="0.35"
-                  power="0.5"/>
-
-            <input>
-                <port id="1">
-                    <dim>2</dim>
-                    <dim>3</dim>
-                    <dim>5</dim>
-                    <dim>5</dim>
-                </port>
-            </input>
-            <output>
-                <port id="2">
-                    <dim>2</dim>
-                    <dim>3</dim>
-                    <dim>5</dim>
-                    <dim>5</dim>
-                </port>
-            </output>
-        </layer>
-        <layer name="sum" type="Eltwise" precision="FP32" id="2">
-            <data coeff="0.77,0.33"/>
-            <input>
-                <port id="1">
-                    <dim>2</dim>
-                    <dim>3</dim>
-                    <dim>5</dim>
-                    <dim>5</dim>
-                </port>
-                <port id="2">
-                    <dim>2</dim>
-                    <dim>3</dim>
-                    <dim>5</dim>
-                    <dim>5</dim>
-                </port>
-            </input>
-            <output>
-                <port id="3">
-                    <dim>2</dim>
-                    <dim>3</dim>
-                    <dim>5</dim>
-                    <dim>5</dim>
-                </port>
-            </output>
-        </layer>
-    </layers>
-    <edges>
-        <edge from-layer="0" from-port="0" to-layer="1" to-port="1"/>
-        <edge from-layer="1" from-port="2" to-layer="2" to-port="1"/>
-        <edge from-layer="0" from-port="0" to-layer="2" to-port="2"/>
-    </edges>
-    <pre-process reference-layer-name="data">
-        <channel id="0">
-            <mean value="104.006"/>
-            <scale value="0.1"/>
-        </channel>
-        <channel id="1">
-            <mean value="116.668"/>
-            <scale value="0.2"/>
-        </channel>
-        <channel id="2">
-            <mean value="122.678"/>
-            <scale value="0.3"/>
```

Comment:
we need IR with floating point values while new V10 version does not have it

Reply:
added